src/Export_Command.php

```diff
@@ -118,35 +122,54 @@ public function __invoke( $_, $assoc_args ) {
 			'filename_format'   => '{site}.wordpress.{date}.{n}.xml',
 		);
 
+
+		if (! empty( $assoc_args['stdout'] ) && ( ! empty( $assoc_args['dir'] ) || ! empty( $assoc_args['filename_format'] ) ) ) {
+			WP_CLI::error( "--stdout and --dir cannot be used together." );
+			WP_CLI::halt(1);
```

Comment:
`WP_CLI::halt(1)` isn't necessary here, because `WP_CLI::error()` will halt the process.